README.md

```diff
@@ -39,7 +54,7 @@ Here are the available instructions:
 
 ## Compatibility
 
-Delight is compatible with Spark `2.4.0` to Spark `3.0.1` with the following Maven coordinates:
+Delight is compatible with Spark `2.4.0` to Spark `3.1.1` with the following Maven coordinates:
 
 `co.datamechanics:delight_<replace-with-your-scala-version-2.11-or-2.12>:latest-SNAPSHOT`
```

Comment:
Is `latest-SNAPSHOT` correct?

Reply:
cc @jrj-d ?

Reply:
Yes it is. That's our way to do continuous deployment for users on Delight.
(We cannot remove the `SNAPSHOT` because of how Maven works unfortunately)